moban/engine.py

```diff
@@ -69,19 +44,17 @@ def __init__(self, template_dirs, context_dirs):
             trim_blocks=True,
             lstrip_blocks=True,
         )
-        for filter_name, filter_function in _FILTERS.get_all():
+        for filter_name, filter_function in FILTERS.get_all():
             self.jj2_environment.filters[filter_name] = filter_function
 
-        for test_name, test_function in _TESTS.get_all():
+        for test_name, test_function in TESTS.get_all():
             self.jj2_environment.tests[test_name] = test_function
 
-        for global_name, dict_obj in _GLOBALS.get_all():
+        for global_name, dict_obj in GLOBALS.get_all():
             self.jj2_environment.globals[global_name] = dict_obj
 
         self.context = Context(context_dirs)
         self.template_dirs = template_dirs
-        self.__file_count = 0
-        self.__templated_count = 0
 
     def render_to_file(self, template_file, data_file, output_file):
```

Comment:
is it possible to implement this function with _apply_template below? This function duplicates some of the code. how would approach it? The ideal end result is: less code to do the same thing.

Reply:
Thanks for taking care of this before I got to it!